framework/doc/content/source/mfem/problem/MFEMProblem.md

```diff
@@ -0,0 +1,30 @@
+# MFEMProblem
+
+## Summary
+
+!syntax description /Problem/MFEMProblem
+
+## Overview
+
+`MFEMProblem` is derived from MOOSE's `ExternalProblem` Problem type, customised to set up the
+ finite element problem using the MFEM FE library instead of MOOSE's default libMesh. Use of MFEM
+ allows problem assembly and solution on GPU architectures as well as on CPU; desired device can be
+ controlled by the [`MFEMExecutioner`](MFEMExecutioner.md) used to solve the problem.
+
+`MFEMProblem` methods are called by `Actions` during parsing of the user's input file, which add
+ and/or initialize members of the owned [MFEMProblemData](source/problem/MFEMProblemData.md) struct.
+The order in which these actions are executed respects the dependencies declared in
+ [PlatypusApp](source/base/PlatypusApp.md).
```

Comment:
All documentation paths for docs from Platypus including `source` should update to `source/mfem` - documentation is currently failing to create these links during build